pyproject.toml

```diff
@@ -77,8 +77,3 @@ showcontent = false
 
 [tool.black]
 line-length = 120
```

Comment:
We should also be able to remove the following section now we've eliminated the SPDX report: https://github.com/ARMmbed/mbed-tools/blob/d3c251d0d43b2aba272aff73c6e152adfef86380/pyproject.toml#L25

Reply:
Removed